src/XMakeBuildEngine/Evaluation/Evaluator.cs

```diff
@@ -2307,7 +2355,13 @@ private List<ProjectRootElement> ExpandAndLoadImports(string directoryOfImportin
                 return LoadImportsResult.ConditionWasFalse;
             }
 
-            return ExpandAndLoadImportsFromUnescapedImportExpression(directoryOfImportingFile, importElement, importElement.Project, throwOnFileNotExistsError, out projects);
+            string project = importElement.Project;
+            if (!String.IsNullOrWhiteSpace(importElement.Sdk))
+            {
+                project = Path.Combine(BuildEnvironmentHelper.Instance.MSBuildSDKsPath, importElement.Sdk, "Sdk", project);
```

Comment:
This logic is duplicated. I guess that ultimately this bit of code would use an IAsserResolver and a potential SDKProjectElement to do its business.